examples/http_config.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
@eager you have any suggestion on the best place to keep things like this config file?  Ansible?  @jeffreyzupka suggested checking them into dunlop someplace.